core/src/test/java/nl/nn/adapterframework/logging/TestLogMessages.java

```diff
@@ -165,6 +165,24 @@ public void testUnicodeCharactersInMessage() {
 		}
 	}
 
+	@Test
+	public void testHideRegexMatchInMessage() {
+		TestAppender appender = TestAppender.newBuilder().useIbisPatternLayout(PATTERN).build();
+		TestAppender.addToRootLogger(appender);
+		try {
+			log.debug("my beautiful log with <password>TO BE HIDDEN</password> hidden value");
+
+			List<String> logEvents = appender.getLogLines();
+			assertEquals(1, logEvents.size(), "found messages "+logEvents);
+			String message = logEvents.get(0);
+			assertEquals("DEBUG - my beautiful log with <password>************</password> hidden value", message);
```

Comment:
Log something like what?

Anyway the ticket was to restore some functionality that got accidentally lost and this is how it used to work. I have not changed the workings of the "hide" function.

Reply:
Ok, I see. Ah... I now see my comment was corrupted or it was hiding my real suggestion, between brackets :-) 
I meant: ...something like `<hidden>` instead?

Reply:
Well, apart from it being functionality-as-is being restored, I think using `*` to hide your password, credit-card details, etc is common.
It also doesn't need to be machine-parseable. It's just the default regex was made to check for passwords in XML tags, presumably because at some point in the past, some XML messages were being logged that contained username/pwd tags.

Reply:
Yeah, we can leave it as is. If we would design it as new, I'd to for another pattern in the logs instead.